packages/grafana-ui/src/components/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { Pagination } from './Pagination';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+const getPageButton = async (page: number) => {
+  return await screen.queryByText(`${page}`);
+};
+const getNextButton = async () => {
+  return await screen.getByLabelText('next');
+};
+const getPreviousButton = async () => {
+  return await screen.getByLabelText('previous');
+};
+const getAllPaginationElements = async () => {
+  return await screen.findAllByRole('listitem');
+};
```

Comment:
```suggestion
const getPageButton = (page: number) => {
  return screen.queryByText(`${page}`);
};
const getNextButton = () => {
  return screen.getByLabelText('next');
};
const getPreviousButton = () => {
  return screen.getByLabelText('previous');
};
const getAllPaginationElements = () => {
  return screen.findAllByRole('listitem');
};
```
No need to use `asyn/await` if the only thing the function does is return a promise. Plus, only `find*` queries return promises.